### DISCUSSION TITLE
HA picture elements with the shower picture ?

### DISCUSSION THREAD

**Discussion by lars010101** [raomin/ESPAltherma#383] [Q&A]
Hi, thanks for this great project! I managed to get up and running with ESP01 board (will post some info later). But as I try set it up in Home Assistant .. I cant find that nice picture showed on the project github page. The one with the showering dude, and the heat pump etc with temp values.

Where can I find that picture without any temp values in it so I can use it to build my HA dashboard ?

**Comment by thehijacker** [comment]
I found it on my HA cards. Not sure where I got it. Probably from this site somehwere :). Still use it today:

![image](https://github.com/raomin/ESPAltherma/assets/13857043/97253ae8-6b06-4d13-81e6-e5cd924f7bb4)

Original image:

![altherma](https://github.com/raomin/ESPAltherma/assets/13857043/aa8e0f8e-c5ac-4a87-8a2c-957e76914175)
